src/QuickMarcEditor/QuickMarcEditorRows/QuickMarcEditorRows.js

```diff
@@ -112,6 +112,12 @@ const QuickMarcEditorRows = ({
     }
   }, [indexOfNewRow, newRowRef]);
 
+  const processSubfieldFocus = useCallback(({ target }) => {
```

Comment:
I think this function can be defined in ContentField itself so we don't have to pass it as a prop

Reply:
Thanks, fixed